lms/djangoapps/courseware/masquerade.py

```diff
@@ -71,27 +72,27 @@ def handle_ajax(request, course_key_string):
     group_id = request_json.get('group_id', None)
     user_partition_id = request_json.get('user_partition_id', None) if group_id is not None else None
     user_name = request_json.get('user_name', None)
+    found_user_name = None
+    # This view should return an error if no 'user_name' was provided, but many different test cases
+    # rely on the view still working if 'user_name' is blank or None.
```

Comment:
@amir-qayyum-khan I'm not sure what this comment means.

`found_user_name = None` statement seems redundant to me, please remove it and the comment below it.